submissions/nilamop/popup/index.html

```diff
@@ -0,0 +1,154 @@
+                                    <a href="#">
+                                        <img src="./img/files.png" alt="files">
+                                        <span>Files</span>
+                                    </a>
+                                </li>
+                                <li class="popup_menu_item">
+                                    <a href="#">
+                                        <img src="./img/contacts.png" alt="contacts">
+                                        <span>Contacts</span>
+                                    </a>
+                                </li>
+                                <li class="popup_menu_item">
+                                    <a href="#">
+                                        <img src="./img/tanslation.png" alt="tanslation">
+                                        <span>Tanslation</span>
+                                    </a>
+                                </li>
+                                <li class="popup_menu_item">
+                                    <a href="#">
+                                        <img src="./img/videos.png" alt="videos">
+                                        <span>Videos</span>
+                                    </a>
+                                </li>
+                                <li class="popup_menu_item">
+                                    <a href="#">
+                                        <img src="./img/search.png" alt="search">
+                                        <span>Search</span>
+                                    </a>
+                                </li>
+                                <li class="popup_menu_item">
+                                    <a href="#">
+                                        <img src="./img/photos.png" alt="photos">
+                                        <span>Photos</span>
+                                    </a>
+                                </li>
+                                <li class="popup_menu_item">
+                                    <a href="#">
+                                        <img src="./img/apps.png" alt="apps">
+                                        <span>Apps</span>
+                                    </a>
+                                </li>
+                                <li class="popup_menu_item">
+                                    <a href="#">
+                                        <img src="./img/documents.png" alt="documents">
+                                        <span>Documents</span>
+                                    </a>
+                                </li>
+                            </ul>
+                            <input id="menu_input_more" class="popup_menu_more" type="checkbox">
+                            <label for="menu_input_more" class="lable__more">
+                                <span>More</span>
+                            </label>
```

Comment:
Add styles on focus state. Users must have the ability to interact with this button using keyboard.

Reply:
Still not fixed :cry: